partiql-logical/src/lib.rs

```diff
@@ -142,6 +142,11 @@ where
         &self.nodes
     }
 
+    /// Returns the operators of the plan.
```

Comment:
nit: comment is the same as for `operators`. could include that the returned iterator includes the op id
```suggestion
    /// Returns the operators of the plan with their `OpId`
```